cdm/src/test/java/ucar/nc2/ft/simpgeometry/TestCFSimpleGeometryCont.java

```diff
@@ -0,0 +1,319 @@
+package ucar.nc2.ft.simpgeometry;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import ucar.nc2.ft2.coverage.simpgeometry.*;
+
+import java.util.Random;
+import java.util.List;
+
+/**
+ * Very simple tests for Simple Geometry container objects.
+ * 
+ * @author wchen@usgs.gov
+ *
+ */
+public class TestCFSimpleGeometryCont {
+	
+	private static final double delt = 0.00;
+	private static final int testsize = 100000;

```

Comment:
How is test 100 different from test 101?
What is the smallest possible test size that addresses corner cases?

Reply:
Yeah I'll address corner cases, thanks! And split them up.